uni/assets/text/TermsAndConditions.md

```diff
@@ -1,138 +1,138 @@
+By using your faculty account in the Mobile Application, you are responsible for maintaining the
 security of your account and you are fully responsible for all activities that occur under the
 account and any other actions taken in connection with it. we will not be liable for any acts or
 omissions by you, including any damages of any kind incurred as a result of such acts or omissions.
-any interaction with your data from universidade do porto's platform is made directly through it,
+Any interaction with your data from Universidade do Porto's platform is made directly through it,
 and is not collected by us in any way.
-if you choose to select the option to keep you logged in, you understand that your sign-in details
-may be stored locally, but not in any way shared with us. these details are stored in the
+If you choose to select the option to keep you logged in, you understand that your sign-in details
+may be stored locally, but not in any way shared with us. These details are stored in the
 application data and we hold no responsibility for ensuring their concealment.
 \
 \
-**ownership of content**
+**Ownership of Content**
 
-we do not own any data, information or material ("content") that you submit in the mobile
-application in the course of using the service. you shall have sole responsibility for the accuracy,
+We do not own any data, information or material ("Content") that you submit in the Mobile
+Application in the course of using the Service.You shall have sole responsibility for the accuracy,
 quality, integrity, legality, reliability, appropriateness, and intellectual property ownership or
-right to use of all submitted content.
+right to use of all submitted Content.
 \
 \
-**data usage, monitoring and privacy**
+**Data usage, Monitoring and Privacy**
 
-we may, at our discretion but with no obligation, monitor and review content in the mobile
-application submitted or created using our services by you.
-personal data of users is not collected for any other purpose, unless providing the services to you,
+We may, at our discretion but with no obligation, monitor and review Content in the Mobile
+Application submitted or created using our Services by you.
+Personal data of users is not collected for any other purpose, unless providing the Services to you,
 except in certain circumstances described
-in [play store’ privacy policy](https://support.google.com/googleplay/android-developer/answer/10144311?visit_id=638365485539535125-3072678242&rd=1)
-and [app store's privacy policy](https://developer.apple.com/app-store/app-privacy-details/#user-tracking).
+in [Play Store’ Privacy Policy](https://support.google.com/googleplay/android-developer/answer/10144311?visit_id=638365485539535125-3072678242&rd=1)
+and [App Store's Privacy Policy](https://developer.apple.com/app-store/app-privacy-details/#user-tracking).
```

Comment:
I think "Play Store's Privacy Policy" and "App Store's Privacy Policy" should have consistent punctuation (?).